src/app/components/shell/header/header.scss

```diff
@@ -19,6 +38,17 @@
         background-color: transparent;
     }
 
+    //zach class styles
+    &.active {
+      @include small {
+        right: 0;
+        bottom: 0;
+        width:100%;
+        height:100%;
+        background: #fff;
```

Comment:
Try to use the global color variables.  This should be `background: color(neutral-lightest);`.